modules/api/php/endpoints/candidate/visit/image/format/raw.class.inc

```diff
@@ -113,7 +113,7 @@ class Raw extends Endpoint implements \LORIS\Middleware\ETagCalculator
         }
 
         $mincpath = \NDB_Factory::singleton()->config()
-            ->getSetting('MINCToolsPath');
+            ->getSetting('MINCToolsPath') ?? '';
```

Comment:
I wouldn't depend on the behaviour of SPLFileInfo with an empty string since I can't find any documentation of what the behaviour is. You should explicitly throw an exception if null or empty.